verification/curator-service/ui/src/index.tsx

```diff
@@ -10,6 +10,13 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import store from './redux/store';
 import { Provider } from 'react-redux';
+import TagManager from 'react-gtm-module';
+
+const tagManagerArgs = {
+    gtmId: 'GTM-WHCJVVH',
```

Comment:
Checking that this is fine to be public @maciekz1996 ?

Reply:
Yes, according to [the setup guide](https://support.google.com/tagmanager/answer/6103696?hl=en#zippy=%2Cweb-pages) you put a JS script with your tag ID into the published HTML, and I can see other sites with their GTM ID published too, this is fine.